ilastik/utility/exportFile.py

```diff
@@ -243,12 +245,34 @@ def _slice_range(low, high, max_):
     return slice(start, end, None)
 
 
-def create_slicing(axistags, dimensions, margin, feature_table):
+def create_slicing(
+    axistags: AxisTags, dimensions: Tuple[int, int, int, int, int], margin: int, feature_table: np.ndarray
+) -> Iterator[Tuple[List[slice], int]]:
     """
-    Returns an iterator on the slices for each object roi
+    Returns an iterator on the slices for each object (bounding box + margin)
         yields also the actual object id
+
+    Args:
+        axistags: image axis tags
+        dimensions: 5D image dimensions in (t, x, y, z, c) -order
+        margin: how much "halo" around each object's bounding box
+        feature_table: structured array with at least the following fields:
```

Comment:
Maybe move schema description from parameter doc into the function doc? In this case, the description can be properly formatted with definition lists:

```
``feature_table`` should have the following fields:

``labelimage_oid``
    object ids, reset per timeframe
``Bounding Box Minimum_0``
    minimum in x-direction

``feature_table`` may also have the following fields:

``Bounding Box Minimum_2``
    minimum in z-direction
```